tests/test_lab_init.py

```diff
@@ -22,26 +18,54 @@ class TestLabInit:
     @patch("instructlab.config.init.Repo.clone_from")
     def test_init_noninteractive(self, mock_clone_from):
         runner = CliRunner()
-        with runner.isolated_filesystem():
+        with runner.isolated_filesystem() as td:
+            os.environ["HOME"] = td
```

Comment:
Similar problem ... I'll get back to you with a proper fixture.

Reply:
Please refactor the code to use `cli_runner`. It will automatically mock patch `os.environ` and reset it at the end of the test.